### ISSUE TITLE
No module named 'work'

### ISSUE THREAD

**Issue by dlhagger** [talmolab/sleap#1803]
# Bug description

Fresh install of sleap 1.4.1a1 via mamba (source) and now I am getting this error when I call sleap-label

Error occurred during callback 0 for color predicted!
[<built-in method setChecked of PySide2.QtWidgets.QAction object at 0x16dc00c80>, <function MainWindow._load_overlays.<locals>.overlay_state_connect.<locals>.<lambda> at 0x16dc7e160>, <bound method MainWindow.plotFrame of <sleap.gui.app.MainWindow(0x60000246da40) at 0x313fa8dc0>>]
No module named 'work'
Restoring GUI state...

## Your personal set up

- OS:

- Version(s):
MacOS 14.5

- SLEAP installation method (listed [here](https://sleap.ai/installation.html#)):
  - [ ] [Conda from package](https://sleap.ai/installation.html#conda-package)
  - [ ] [Conda from source](https://sleap.ai/installation.html#conda-from-source)
  - [ ] [pip package](https://sleap.ai/installation.html#pip-package)
  - [X ] [Apple Silicon Macs](https://sleap.ai/installation.html#apple-silicon-macs)

<details><summary>Environment packages</summary>

```
# paste output of `pip freeze` or `conda list` here
```
absl-py                   2.1.0                    pypi_0    pypi
accessible-pygments       0.0.5                    pypi_0    pypi
alabaster                 0.7.16                   pypi_0    pypi
albumentations            1.3.1              pyhd8ed1ab_0    conda-forge
anyio                     3.7.1                    pypi_0    pypi
aom                       3.6.1                hb765f3a_0    conda-forge
appdirs                   1.4.4                    pypi_0    pypi
appnope                   0.1.4                    pypi_0    pypi
argon2-cffi               23.1.0                   pypi_0    pypi
argon2-cffi-bindings      21.2.0                   pypi_0    pypi
asttokens                 2.4.1                    pypi_0    pypi
astunparse                1.6.3                    pypi_0    pypi
attrs                     21.4.0                   pypi_0    pypi
babel                     2.15.0                   pypi_0    pypi
backports-tarfile         1.2.0                    pypi_0    pypi
beautifulsoup4            4.12.3                   pypi_0    pypi
black                     21.6b0                   pypi_0    pypi
blas                      2.120                  openblas    conda-forge
blas-devel                3.9.0           20_osxarm64_openblas    conda-forge
bleach                    6.1.0                    pypi_0    pypi
blosc                     1.21.5               hc338f07_0    conda-forge
brotli                    1.0.9                h1a8c8d9_9    conda-forge
brotli-bin                1.0.9                h1a8c8d9_9    conda-forge
brunsli                   0.1                  h9f76cd9_0    conda-forge
bzip2                     1.0.8                h93a5062_5    conda-forge
c-ares                    1.28.1               h93a5062_0    conda-forge
c-blosc2                  2.12.0               ha57e6be_0    conda-forge
ca-certificates           2024.6.2             hf0a4a13_0    conda-forge
cached-property           1.5.2                hd8ed1ab_1    conda-forge
cached_property           1.5.2              pyha770c72_1    conda-forge
cachetools                5.3.3                    pypi_0    pypi
cairo                     1.18.0               hd1e100b_0    conda-forge
cattrs                    1.1.1              pyhd8ed1ab_0    conda-forge
certifi                   2024.6.2           pyhd8ed1ab_0    conda-forge
cffi                      1.16.0                   pypi_0    pypi
cfgv                      3.4.0                    pypi_0    pypi
charls                    2.4.2                h13dd4ca_0    conda-forge
charset-normalizer        3.3.2                    pypi_0    pypi
click                     8.0.4                    pypi_0    pypi
cloudpickle               3.0.0              pyhd8ed1ab_0    conda-forge
colorama                  0.4.6                    pypi_0    pypi
comm                      0.2.2                    pypi_0    pypi
contourpy                 1.2.1            py39h48c5dd5_0    conda-forge
coverage                  7.5.3                    pypi_0    pypi
cryptography              42.0.8                   pypi_0    pypi
cycler                    0.12.1             pyhd8ed1ab_0    conda-forge
cytoolz                   0.12.3           py39h17cfd9d_0    conda-forge
dask-core                 2024.5.2           pyhd8ed1ab_0    conda-forge
dav1d                     1.2.1                hb547adb_0    conda-forge
debugpy                   1.8.1                    pypi_0    pypi
decorator                 5.1.1                    pypi_0    pypi
defusedxml                0.7.1                    pypi_0    pypi
deprecated                1.2.14                   pypi_0    pypi
distlib                   0.3.8                    pypi_0    pypi
docutils                  0.17.1                   pypi_0    pypi
efficientnet              1.0.0                    pypi_0    pypi
entrypoints               0.4                      pypi_0    pypi
exceptiongroup            1.2.1                    pypi_0    pypi
executing                 2.0.1                    pypi_0    pypi
expat                     2.6.2                hebf3989_0    conda-forge
ffmpeg                    6.1.0           gpl_h85ac7dd_100    conda-forge
filelock                  3.14.0                   pypi_0    pypi
flatbuffers               1.12                     pypi_0    pypi
font-ttf-dejavu-sans-mono 2.37                 hab24e00_0    conda-forge
font-ttf-inconsolata      3.000                h77eed37_0    conda-forge
font-ttf-source-code-pro  2.038                h77eed37_0    conda-forge
font-ttf-ubuntu           0.83                 h77eed37_2    conda-forge
fontconfig                2.14.2               h82840c6_0    conda-forge
fonts-conda-ecosystem     1                             0    conda-forge
fonts-conda-forge         1                             0    conda-forge
fonttools                 4.53.0           py39hfea33bf_0    conda-forge
freetype                  2.12.1               hadb7bae_2    conda-forge
fribidi                   1.0.10               h27ca646_0    conda-forge
fsspec                    2024.6.0           pyhff2d567_0    conda-forge
gast                      0.4.0                    pypi_0    pypi
gettext                   0.22.5               h8fbad5d_2    conda-forge
gettext-tools             0.22.5               h8fbad5d_2    conda-forge
giflib                    5.2.2                h93a5062_0    conda-forge
gitdb                     4.0.11                   pypi_0    pypi
gitpython                 3.1.43                   pypi_0    pypi
glib                      2.80.2               h535f939_0    conda-forge
glib-tools                2.80.2               h4c882b9_0    conda-forge
gmp                       6.3.0                hebf3989_1    conda-forge
gnutls                    3.7.9                hd26332c_0    conda-forge
google-auth               2.30.0                   pypi_0    pypi
google-auth-oauthlib      0.4.6                    pypi_0    pypi
google-pasta              0.2.0                    pypi_0    pypi
graphite2                 1.3.13            hebf3989_1003    conda-forge
grpcio                    1.64.1                   pypi_0    pypi
gst-plugins-base          1.22.9               h09b4b5e_1    conda-forge
gstreamer                 1.22.9               h551c6ff_1    conda-forge
h11                       0.14.0                   pypi_0    pypi
h5py                      3.9.0           nompi_py39he9c2634_101    conda-forge
harfbuzz                  7.3.0                hf1a6348_1    conda-forge
hdf5                      1.14.1          nompi_h3aba7b3_100    conda-forge
hdmf                      3.14.1             pyh2e8e312_0    conda-forge
icu                       73.2                 hc8870d7_0    conda-forge
identify                  2.5.36                   pypi_0    pypi
idna                      3.7                      pypi_0    pypi
image-classifiers         1.0.0                    pypi_0    pypi
imagecodecs               2023.8.12        py39h24b8818_0    conda-forge
imageio                   2.34.1             pyh4b66e23_0    conda-forge
imagesize                 1.4.1                    pypi_0    pypi
imgstore                  0.2.9                    pypi_0    pypi
importlib-metadata        7.0.2              pyha770c72_0    conda-forge
importlib-resources       6.4.0              pyhd8ed1ab_0    conda-forge
importlib_metadata        7.0.2                hd8ed1ab_0    conda-forge
importlib_resources       6.4.0              pyhd8ed1ab_0    conda-forge
iniconfig                 2.0.0                    pypi_0    pypi
ipykernel                 6.29.4                   pypi_0    pypi
ipython                   8.18.1                   pypi_0    pypi
ipython-genutils          0.2.0                    pypi_0    pypi
ipywidgets                7.8.1                    pypi_0    pypi
jaraco-classes            3.4.0                    pypi_0    pypi
jaraco-context            5.3.0                    pypi_0    pypi
jaraco-functools          4.0.1                    pypi_0    pypi
jasper                    4.0.0                hff9eb24_1    conda-forge
jedi                      0.17.2                   pypi_0    pypi
jinja2                    3.1.4                    pypi_0    pypi
joblib                    1.4.2              pyhd8ed1ab_0    conda-forge
jsmin                     3.0.1              pyhd8ed1ab_0    conda-forge
json5                     0.9.25                   pypi_0    pypi
jsonpickle                1.2                        py_0    conda-forge
jsonschema                4.17.3                   pypi_0    pypi
jsonschema-specifications 2023.12.1          pyhd8ed1ab_0    conda-forge
jupyter-cache             0.4.3                    pypi_0    pypi
jupyter-client            7.4.9                    pypi_0    pypi
jupyter-core              5.7.2                    pypi_0    pypi
jupyter-server            1.13.5                   pypi_0    pypi
jupyter-server-mathjax    0.2.6                    pypi_0    pypi
jupyter-sphinx            0.3.2                    pypi_0    pypi
jupyterlab                3.3.4                    pypi_0    pypi
jupyterlab-pygments       0.3.0                    pypi_0    pypi
jupyterlab-server         2.16.3                   pypi_0    pypi
jupyterlab-widgets        1.1.7                    pypi_0    pypi
jxrlib                    1.1                  h93a5062_3    conda-forge
keras                     2.9.0              pyhd8ed1ab_0    conda-forge
keras-applications        1.0.8                    pypi_0    pypi
keras-preprocessing       1.1.2                    pypi_0    pypi
keyring                   25.2.1                   pypi_0    pypi
kiwisolver                1.4.5            py39hbd775c9_1    conda-forge
krb5                      1.21.2               h92f50d5_0    conda-forge
lame                      3.100             h1a8c8d9_1003    conda-forge
lazy_loader               0.4                pyhd8ed1ab_0    conda-forge
lcms2                     2.15                 hd835a16_1    conda-forge
lerc                      4.0.0                h9a09cb3_0    conda-forge
libaec                    1.1.3                hebf3989_0    conda-forge
libasprintf               0.22.5               h8fbad5d_2    conda-forge
libasprintf-devel         0.22.5               h8fbad5d_2    conda-forge
libass                    0.17.1               h4da34ad_0    conda-forge
libavif                   0.11.1               h80987f9_0    anaconda
libblas                   3.9.0           20_osxarm64_openblas    conda-forge
libbrotlicommon           1.0.9                h1a8c8d9_9    conda-forge
libbrotlidec              1.0.9                h1a8c8d9_9    conda-forge
libbrotlienc              1.0.9                h1a8c8d9_9    conda-forge
libcblas                  3.9.0           20_osxarm64_openblas    conda-forge
libclang                  18.1.1                   pypi_0    pypi
libclang13                15.0.7          default_h83d0a53_5    conda-forge
libcurl                   8.8.0                h7b6f9a7_0    conda-forge
libcxx                    17.0.6               h5f092b4_0    conda-forge
libdeflate                1.18                 h1a8c8d9_0    conda-forge
libedit                   3.1.20191231         hc8eb9b7_2    conda-forge
libev                     4.33                 h93a5062_2    conda-forge
libexpat                  2.6.2                hebf3989_0    conda-forge
libffi                    3.4.2                h3422bc3_5    conda-forge
libgettextpo              0.22.5               h8fbad5d_2    conda-forge
libgettextpo-devel        0.22.5               h8fbad5d_2    conda-forge
libgfortran               5.0.0           13_2_0_hd922786_3    conda-forge
libgfortran5              13.2.0               hf226fd6_3    conda-forge
libglib                   2.80.2               h535f939_0    conda-forge
libhwloc                  2.10.0          default_h7685b71_1001    conda-forge
libiconv                  1.17                 h0d3ecfb_2    conda-forge
libidn2                   2.3.7                h93a5062_0    conda-forge
libintl                   0.22.5               h8fbad5d_2    conda-forge
libintl-devel             0.22.5               h8fbad5d_2    conda-forge
libjpeg-turbo             2.1.5.1              hb547adb_1    conda-forge
liblapack                 3.9.0           20_osxarm64_openblas    conda-forge
liblapacke                3.9.0           20_osxarm64_openblas    conda-forge
libllvm15                 15.0.7               h2621b3d_4    conda-forge
libnghttp2                1.58.0               ha4dd798_1    conda-forge
libogg                    1.3.4                h27ca646_1    conda-forge
libopenblas               0.3.25          openmp_h6c19121_0    conda-forge
libopencv                 4.8.0            py39h6a99d3f_0    conda-forge
libopenvino               2023.0.1             h13dd4ca_3    conda-forge
libopenvino-arm-cpu-plugin 2023.0.1             h13dd4ca_3    conda-forge
libopenvino-auto-batch-plugin 2023.0.1             h13dd4ca_3    conda-forge
libopenvino-auto-plugin   2023.0.1             h13dd4ca_3    conda-forge
libopenvino-hetero-plugin 2023.0.1             h13dd4ca_3    conda-forge
libopenvino-ir-frontend   2023.0.1             h13dd4ca_3    conda-forge
libopenvino-onnx-frontend 2023.0.1             h13dd4ca_3    conda-forge
libopenvino-paddle-frontend 2023.0.1             h13dd4ca_3    conda-forge
libopenvino-pytorch-frontend 2023.0.1             h13dd4ca_3    conda-forge
libopenvino-tensorflow-frontend 2023.0.1             h13dd4ca_3    conda-forge
libopenvino-tensorflow-lite-frontend 2023.0.1             h13dd4ca_3    conda-forge
libopus                   1.3.1                h27ca646_1    conda-forge
libpng                    1.6.43               h091b4b1_0    conda-forge
libpq                     15.7                 h0e8a1ed_0    conda-forge
libprotobuf               3.21.12              ha614eb4_2    conda-forge
libsodium                 1.0.18               h27ca646_1    conda-forge
libsqlite                 3.45.3               h091b4b1_0    conda-forge
libssh2                   1.11.0               h7a5bd25_0    conda-forge
libtasn1                  4.19.0               h1a8c8d9_0    conda-forge
libtiff                   4.5.1                h23a1a89_1    conda-forge
libunistring              0.9.10               h3422bc3_0    conda-forge
libvorbis                 1.3.7                h9f76cd9_0    conda-forge
libvpx                    1.13.1               hb765f3a_0    conda-forge
libwebp-base              1.4.0                h93a5062_0    conda-forge
libxcb                    1.15                 hf346824_0    conda-forge
libxml2                   2.12.7               ha661575_1    conda-forge
libxslt                   1.1.39               h223e5b9_0    conda-forge
libzlib                   1.3.1                hfb2fe0b_1    conda-forge
libzopfli                 1.0.3                h9f76cd9_0    conda-forge
linkify-it-py             2.0.3                    pypi_0    pypi
llvm-openmp               18.1.7               hde57baf_0    conda-forge
locket                    1.0.0              pyhd8ed1ab_0    conda-forge
lxml                      5.2.2                    pypi_0    pypi
lz4-c                     1.9.4                hb7217d7_0    conda-forge
markdown                  3.6                      pypi_0    pypi
markdown-it-py            1.1.0                    pypi_0    pypi
markupsafe                2.1.5                    pypi_0    pypi
matplotlib-base           3.8.4            py39h15359f4_2    conda-forge
matplotlib-inline         0.1.7                    pypi_0    pypi
mdit-py-plugins           0.2.8                    pypi_0    pypi
mdurl                     0.1.2              pyhd8ed1ab_0    conda-forge
mistune                   0.8.4                    pypi_0    pypi
more-itertools            10.2.0                   pypi_0    pypi
munkres                   1.1.4              pyh9f0ad1d_0    conda-forge
mypy-extensions           1.0.0                    pypi_0    pypi
mysql-common              8.0.33               hf9e6398_6    conda-forge
mysql-libs                8.0.33               he3dca8b_6    conda-forge
myst-nb                   0.13.2                   pypi_0    pypi
myst-parser               0.15.2                   pypi_0    pypi
nbclassic                 0.5.6                    pypi_0    pypi
nbclient                  0.5.13                   pypi_0    pypi
nbconvert                 6.5.4                    pypi_0    pypi
nbdime                    4.0.1                    pypi_0    pypi
nbformat                  5.1.3                    pypi_0    pypi
ncurses                   6.5                  hb89a1cb_0    conda-forge
ndx-pose                  0.1.1              pyhd8ed1ab_0    conda-forge
nest-asyncio              1.6.0                    pypi_0    pypi
nettle                    3.9.1                h40ed0f5_0    conda-forge
networkx                  3.2.1              pyhd8ed1ab_0    conda-forge
nh3                       0.2.17                   pypi_0    pypi
nixio                     1.5.3                    pypi_0    pypi
nodeenv                   1.9.1                    pypi_0    pypi
notebook                  6.5.7                    pypi_0    pypi
notebook-shim             0.2.4                    pypi_0    pypi
nspr                      4.35                 hb7217d7_0    conda-forge
nss                       3.100                hc6e9f88_0    conda-forge
numpy                     1.24.4                   pypi_0    pypi
oauthlib                  3.2.2                    pypi_0    pypi
openblas                  0.3.25          openmp_h55c453e_0    conda-forge
opencv                    4.8.0            py39hdf13c20_0    conda-forge
openh264                  2.3.1                hb7217d7_2    conda-forge
openjpeg                  2.5.0                hbc2ba62_2    conda-forge
openssl                   3.3.1                hfb2fe0b_0    conda-forge
opt-einsum                3.3.0                    pypi_0    pypi
p11-kit                   0.24.1               h29577a5_0    conda-forge
packaging                 24.0               pyhd8ed1ab_0    conda-forge
pandas                    2.2.2            py39h998126f_1    conda-forge
pandocfilters             1.5.1                    pypi_0    pypi
parso                     0.7.1                    pypi_0    pypi
partd                     1.4.2              pyhd8ed1ab_0    conda-forge
pathspec                  0.12.1                   pypi_0    pypi
patsy                     0.5.6              pyhd8ed1ab_0    conda-forge
pcre2                     10.43                h26f9a81_0    conda-forge
pexpect                   4.9.0                    pypi_0    pypi
pillow                    10.0.0           py39h1641143_0    conda-forge
pip                       24.0               pyhd8ed1ab_0    conda-forge
pixman                    0.43.4               hebf3989_0    conda-forge
pkginfo                   1.11.1                   pypi_0    pypi
pkgutil-resolve-name      1.3.10             pyhd8ed1ab_1    conda-forge
platformdirs              4.2.2                    pypi_0    pypi
pluggy                    1.5.0                    pypi_0    pypi
pre-commit                3.7.1                    pypi_0    pypi
prometheus-client         0.20.0                   pypi_0    pypi
prompt-toolkit            3.0.47                   pypi_0    pypi
protobuf                  3.19.6                   pypi_0    pypi
psutil                    5.9.8            py39h17cfd9d_0    conda-forge
pthread-stubs             0.4               h27ca646_1001    conda-forge
ptyprocess                0.7.0                    pypi_0    pypi
pugixml                   1.13                 h13dd4ca_1    conda-forge
pure-eval                 0.2.2                    pypi_0    pypi
py-opencv                 4.8.0            py39h900ff78_0    conda-forge
pyasn1                    0.6.0                    pypi_0    pypi
pyasn1-modules            0.4.0                    pypi_0    pypi
pycparser                 2.22                     pypi_0    pypi
pydata-sphinx-theme       0.13.3                   pypi_0    pypi
pygithub                  2.3.0                    pypi_0    pypi
pygments                  2.18.0             pyhd8ed1ab_0    conda-forge
pyjwt                     2.8.0                    pypi_0    pypi
pykalman                  0.9.7              pyhd8ed1ab_0    conda-forge
pynacl                    1.5.0                    pypi_0    pypi
pynwb                     2.8.0              pyh2e8e312_0    conda-forge
pyparsing                 3.1.2              pyhd8ed1ab_0    conda-forge
pyrsistent                0.20.0                   pypi_0    pypi
pyside2                   5.15.8           py39hdd1c371_4    conda-forge
pytest                    8.2.2                    pypi_0    pypi
pytest-cov                3.0.0                    pypi_0    pypi
pytest-qt                 4.4.0                    pypi_0    pypi
pytest-xvfb               3.0.0                    pypi_0    pypi
python                    3.9.19          hd7ebdb9_0_cpython    conda-forge
python-dateutil           2.9.0              pyhd8ed1ab_0    conda-forge
python-rapidjson          1.17             py39hbf7db11_0    conda-forge
python-tzdata             2024.1             pyhd8ed1ab_0    conda-forge
python_abi                3.9                      4_cp39    conda-forge
pytz                      2024.1             pyhd8ed1ab_0    conda-forge
pyvirtualdisplay          3.0                      pypi_0    pypi
pywavelets                1.4.1            py39h4d8bf0d_0    conda-forge
pyyaml                    6.0.1            py39h0f82c59_1    conda-forge
pyzmq                     26.0.3           py39he7f0319_0    conda-forge
qimage2ndarray            1.10.0                   pypi_0    pypi
qt-main                   5.15.8              h219f738_16    conda-forge
qtpy                      2.4.1              pyhd8ed1ab_0    conda-forge
qudida                    0.0.4              pyhd8ed1ab_0    conda-forge
readline                  8.2                  h92ec313_1    conda-forge
readme-renderer           43.0                     pypi_0    pypi
referencing               0.35.1             pyhd8ed1ab_0    conda-forge
regex                     2024.5.15                pypi_0    pypi
requests                  2.32.3                   pypi_0    pypi
requests-oauthlib         2.0.0                    pypi_0    pypi
requests-toolbelt         1.0.0                    pypi_0    pypi
rfc3986                   2.0.0                    pypi_0    pypi
rich                      13.7.1             pyhd8ed1ab_0    conda-forge
rpds-py                   0.18.1           py39h0019b8a_0    conda-forge
rsa                       4.9                      pypi_0    pypi
ruamel.yaml               0.18.6           py39h17cfd9d_0    conda-forge
ruamel.yaml.clib          0.2.8            py39h17cfd9d_0    conda-forge
scikit-image              0.20.0           py39hd28f0be_1    conda-forge
scikit-learn              1.0              py39h12ba089_1    conda-forge
scikit-video              1.1.11             pyh24bf2e0_0    conda-forge
scipy                     1.9.0            py39h14896cb_0    conda-forge
seaborn                   0.13.2               hd8ed1ab_2    conda-forge
seaborn-base              0.13.2             pyhd8ed1ab_2    conda-forge
segmentation-models       1.0.1                    pypi_0    pypi
send2trash                1.8.3                    pypi_0    pypi
setuptools                70.0.0             pyhd8ed1ab_0    conda-forge
six                       1.15.0                   pypi_0    pypi
sleap                     1.4.1a1                   dev_0    <develop>
smmap                     5.0.1                    pypi_0    pypi
snappy                    1.1.10               hd04f947_1    conda-forge
sniffio                   1.3.1                    pypi_0    pypi
snowballstemmer           2.2.0                    pypi_0    pypi
soupsieve                 2.5                      pypi_0    pypi
sphinx                    4.5.0                    pypi_0    pypi
sphinx-autobuild          2024.4.16                pypi_0    pypi
sphinx-book-theme         1.0.1                    pypi_0    pypi
sphinx-copybutton         0.5.2                    pypi_0    pypi
sphinx-togglebutton       0.3.2                    pypi_0    pypi
sphinxcontrib-applehelp   1.0.8                    pypi_0    pypi
sphinxcontrib-devhelp     1.0.6                    pypi_0    pypi
sphinxcontrib-htmlhelp    2.0.5                    pypi_0    pypi
sphinxcontrib-jsmath      1.0.1                    pypi_0    pypi
sphinxcontrib-qthelp      1.0.7                    pypi_0    pypi
sphinxcontrib-serializinghtml 1.1.10                   pypi_0    pypi
sqlalchemy                1.4.52                   pypi_0    pypi
stack-data                0.6.3                    pypi_0    pypi
starlette                 0.37.2                   pypi_0    pypi
statsmodels               0.14.2           py39h161d348_0    conda-forge
svt-av1                   1.7.0                hb765f3a_0    conda-forge
tbb                       2021.12.0            h420ef59_1    conda-forge
tensorboard               2.9.1                    pypi_0    pypi
tensorboard-data-server   0.6.1                    pypi_0    pypi
tensorboard-plugin-wit    1.8.1                    pypi_0    pypi
tensorflow-estimator      2.9.0                    pypi_0    pypi
tensorflow-hub            0.12.0                   pypi_0    pypi
tensorflow-macos          2.9.2                    pypi_0    pypi
tensorflow-metal          0.5.0                    pypi_0    pypi
termcolor                 2.4.0                    pypi_0    pypi
terminado                 0.18.1                   pypi_0    pypi
threadpoolctl             3.5.0              pyhc1e730c_0    conda-forge
tifffile                  2024.5.22          pyhd8ed1ab_0    conda-forge
tinycss2                  1.3.0                    pypi_0    pypi
tk                        8.6.13               h5083fa2_1    conda-forge
toml                      0.10.2                   pypi_0    pypi
tomli                     2.0.1                    pypi_0    pypi
toolz                     0.12.1             pyhd8ed1ab_0    conda-forge
tornado                   6.4.1                    pypi_0    pypi
tqdm                      4.66.4                   pypi_0    pypi
traitlets                 5.14.3                   pypi_0    pypi
twine                     3.3.0                    pypi_0    pypi
typing-extensions         4.12.2               hd8ed1ab_0    conda-forge
typing_extensions         4.12.2             pyha770c72_0    conda-forge
tzdata                    2024a                h0c530f3_0    conda-forge
tzlocal                   5.2                      pypi_0    pypi
uc-micro-py               1.0.3                    pypi_0    pypi
unicodedata2              15.1.0           py39h0f82c59_0    conda-forge
urllib3                   2.2.1                    pypi_0    pypi
uvicorn                   0.30.1                   pypi_0    pypi
virtualenv                20.26.2                  pypi_0    pypi
watchfiles                0.22.0                   pypi_0    pypi
wcwidth                   0.2.13                   pypi_0    pypi
webencodings              0.5.1                    pypi_0    pypi
websocket-client          1.8.0                    pypi_0    pypi
websockets                12.0                     pypi_0    pypi
werkzeug                  3.0.3                    pypi_0    pypi
wheel                     0.43.0             pyhd8ed1ab_1    conda-forge
widgetsnbextension        3.6.6                    pypi_0    pypi
wrapt                     1.16.0                   pypi_0    pypi
x264                      1!164.3095           h57fd34a_2    conda-forge
x265                      3.5                  hbc6ce65_3    conda-forge
xorg-libxau               1.0.11               hb547adb_0    conda-forge
xorg-libxdmcp             1.1.3                h27ca646_0    conda-forge
xz                        5.2.6                h57fd34a_0    conda-forge
yaml                      0.2.5                h3422bc3_2    conda-forge
zeromq                    4.3.5                hcc0f68c_4    conda-forge
zfp                       1.0.1                ha8f4885_0    conda-forge
zipp                      3.19.2             pyhd8ed1ab_0    conda-forge
zlib                      1.3.1                hfb2fe0b_1    conda-forge
zlib-ng                   2.0.7                h1a8c8d9_0    conda-forge
zstd                      1.5.6                hb46c0d2_0    conda-forge

</details>

<details><summary>Logs</summary>

```
# paste relevant logs here, if any
```

</details>

**Comment by eberrigan** [comment]
@dlhagger, all you have done is typed `sleap-label` in the command line? Thanks!

The related issues are on silicone Macs. I will try to replicate this on another operating system as well.

**Comment by dlhagger** [comment]
That is correct - upon activating the environment and then typing sleap-label, I get the error, but the GUI still opens. 

I tried rolling back pyside2 to and older version, and also co-installing pyside6 to see if it fixed anything, but the issue still remained.

**Comment by roomrys** [comment]
Long awaited, but finally integrated. This issue has been fixed in the SLEAP 1.4.1 release!
